robot_data_sender:
  ros__parameters:
    home_position_x: 0.025
    home_position_y: 0.190
    home_orientation_x: 0.0
    home_orientation_y: 0.0
    home_orientation_z: -0.00416502
    home_orientation_w: 1.0
    home_aruco_id: 9
    robot_type: serving